test/e2e/specs/liquidation-reconstitution.spec.js

```diff
@@ -21,23 +30,226 @@ describe('Wallet App Test Cases', () => {
       }).then(taskCompleted => {
         expect(taskCompleted).to.be.true;
       });
-      cy.setupWallet({
-        secretWords: mnemonics.user1,
-        walletName: 'user1',
-      }).then(taskCompleted => {
+    });
+  });
+
+  context('Adjusting manager params from econ-gov', () => {
+    it('should connect with chain and wallet', () => {
+      cy.visit(econGovURL);
+      cy.acceptAccess().then(taskCompleted => {
         expect(taskCompleted).to.be.true;
       });
     });
+    it('should allow gov2 to create a proposal', () => {
+      cy.visit(econGovURL);
+      cy.acceptAccess();
+
+      cy.get('button').contains('Vaults').click();
+      cy.get('button').contains('Select Manager').click();
+      cy.get('button').contains('manager0').click();
+
+      cy.get('label')
+        .contains('LiquidationMargin')
+        .parent()
+        .within(() => {
+          cy.get('input').clear();
+          cy.get('input').type('150');
+        });
+
+      cy.get('label')
+        .contains('LiquidationPadding')
+        .parent()
+        .within(() => {
+          cy.get('input').clear();
+          cy.get('input').type('25');
+        });
+
+      cy.get('label')
+        .contains('LiquidationPenalty')
+        .parent()
+        .within(() => {
+          cy.get('input').clear();
+          cy.get('input').type('1');
+        });
+
+      cy.get('label')
+        .contains('StabilityFee')
+        .parent()
+        .within(() => {
+          cy.get('input').clear();
+          cy.get('input').type('1');
+        });
+
+      cy.get('label')
+        .contains('MintFee')
+        .parent()
+        .within(() => {
+          cy.get('input').clear();
+          cy.get('input').type('0.5');
+        });
```

Comment:
can this be turned into a custom command? feels like we've used this 100 times in this file